docs/faq.md

```diff
@@ -95,12 +95,12 @@ The method to take a screenshot automatically on failure would depend on the typ
 
 ```javascript
 // Note: this is using Jasmine 1.3 reporter syntax.
```

Comment:
This wasn't intended to be a Jasmine2.0 fix, just to make the code work.  But now that I look more closely, it should work fine as-is.  I thought this code was not working in my local setup, but I didn't really grok the anonymous class that `new function() { ... }` implements.  It actually works fine now that I know how it works.